diff --git a/receivers/electron/forge.config.js b/receivers/electron/forge.config.js
index 8fbbdf9..cb33c83 100644
--- a/receivers/electron/forge.config.js
+++ b/receivers/electron/forge.config.js
@@ -22,6 +22,12 @@ module.exports = {
       appleApiKey: process.env.FCAST_APPLE_API_KEY,
       appleApiKeyId: process.env.FCAST_APPLE_API_KEY_ID,
       appleApiIssuer: process.env.FCAST_APPLE_API_ISSUER
+    },
+    download: {
+      checksums: {
+        "electron-v39.2.7-linux-x64.zip": "2f5285ef563dca154aa247696dddef545d3d895dd9b227ed423ea0d43737c22c",
+        "electron-v39.2.7-linux-arm64.zip": "445465a43bd2ffaec09877f4ed46385065632a4683c2806cc6211cc73c110024"
+      }
     }
   },
   rebuildConfig: {},
